src/yml/normalize_yml.py

```diff
@@ -36,13 +36,20 @@
 Software = list[dict[str, Any]]
 
 
-def munge(filenames: list[str]) -> Software:
+def munge(
+    filenames: list[str], owners: list[dict[str, str]] = [], canonical=False
+) -> Software:
     """Munge together the "software" nodes from YAML files into a single Python dictionary."""
```

Comment:
Update docstring to reflect additional functionality:
```suggestion
    """Munge together the "owners" and "software" nodes from YAML files into a single Python dictionary."""
```